### ISSUE TITLE
PB450 Password Jumper keeps triggering, Locking the System, plus hanging at boot

### ISSUE THREAD

**Issue by J03yPJASDF** [86Box/86Box#4415]
### What happened?

Attempting to enter the BIOS Setup often causes hanging like this:

![Monitor_1_20240430-201309-378](https://github.com/86Box/86Box/assets/166947898/85fe11f8-bcfa-4897-bb69-79eccebed5c3)

Followed by this: 

![Monitor_1_20240430-201354-331](https://github.com/86Box/86Box/assets/166947898/e2416a51-a4b4-4e7b-b194-5761cbedc7a8)

This screen can also come after simply waiting at the bootup screen.

There's also a hanging issue as well. Only way out is a Hard Reset. Chance of this coming back even after that is random.

**Steps to Reproduce:**

1#: Boot the Machine and let the Bootup do it's thing

2#: Wait a while, or press random keys

**OR:**

1#: Press F2 after a software reset as the machine is booting. On a fresh NVR there will be a message about the CMOS being invalid, but that's expected of a fresh setup.

**Expected Result:**

Entering BIOS Setup without any fuss, or just the machine booting up.

### Configuration file

```ini
[General]
vid_renderer = qt_software
host_cpu = Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz
emu_build_num = 5722
uuid = aa61e208-8a41-5981-969e-62ce5d01e726

[Machine]
machine = pb450
cpu_family = i486dx2
cpu_speed = 66666666
cpu_multi = 2
cpu_use_dynarec = 0
fpu_softfloat = 0
time_sync = local
fpu_type = internal
mem_size = 16384

[Video]
gfxcard = internal

[Input devices]
mouse_type = ps2
joystick_type = 4axis_4button
joystick_0_nr = 2
joystick_0_axis_0 = -2147483648
joystick_0_axis_1 = 1073741824
joystick_0_axis_2 = 1
joystick_0_axis_3 = 0
joystick_0_button_0 = 2
joystick_0_button_1 = 3
joystick_0_button_2 = 0
joystick_0_button_3 = 1

[Sound]
fm_driver = nuked
sndcard = azt2316a

[Network]
net_01_link = 0
net_02_link = 0
net_03_link = 0
net_04_link = 0
net_01_card = pcnetpci
net_01_net_type = slirp

[Storage controllers]
hdc = internal
cassette_mode = load

[Standard PS/2 Mouse]
buttons = 3

[AMD PCnet-PCI II #1]
mac = e7:f8:b1

[Ports (COM & LPT)]
lpt1_device = dot_matrix

[Hard disks]
hdd_01_parameters = 63, 16, 1024, 0, ide
hdd_01_fn = LEG20CDTEST.vhd
hdd_01_vhd_blocksize = 1024
hdd_01_speed = ramdisk
hdd_01_ide_channel = 0:0

[Floppy and CD-ROM drives]
fdd_01_type = 35_2hd
fdd_02_type = 525_2hd
cdrom_01_speed = 2
cdrom_01_parameters = 1, atapi
cdrom_01_ide_channel = 0:1
cdrom_01_type = MATSHITA_CR-572_1.0j

[Other removable devices]
zip_01_parameters = 0, atapi
zip_01_ide_channel = 1:0
```

### Operating system

Windows 10 Pro

### CPU

Intel Core i5-7500 @ 3.40 GHz

### 86Box version

build 5722

### Build architecture

Windows - x64 (64-bit)

### Build type

- [X] New recompiler
- [X] Debug build

### Download source

Official website (Jenkins, GitHub)

### Additional context

This also happens on the release build (4.1.1 Build 5634), as well the old Compiler Build of 5722

Test Machine (Includes Cracked PB Restore boot & DOS 6.20 Boot):
[Legend 20CD (PB450) Test.zip](https://github.com/86Box/86Box/files/15172840/Legend.20CD.PB450.Test.zip)

Note that nothing is on the hard disk and it isn't even formatted.

**Comment by randomsomeone69-real** [comment]
I have tested your config and yes the message appears, but not all times. When I pressed the F2 button, yes there is a glitched screen but not all times the password reset jumper. I tried after that to install a french copy of MS-DOS 5. Long story short : I managed but the Intel PnP screen is a pain. This install needs 2 reboots (one for FDISK and one after the install) and at any moment no password jumper screen has showed up. 

![MS-DOS 5](https://github.com/86Box/86Box/assets/160155909/2f6c7954-6867-459d-bbe1-ff549c330cf9)

I also removed some things (joysticks, sound card, ZIP drives and network drives) as I deemed it wasn't necessary in order to boot.

Try remove this too and retry your install as I'm not a developer.
